packages/core/src/common/props.ts

```diff
@@ -86,9 +86,6 @@ export interface IControlledProps {
     /** Initial value of the input, for uncontrolled usage. */
     defaultValue?: string;
 
-    /** Change event handler. Use `event.target.value` for new value. */
-    onChange?: React.FormEventHandler<HTMLElement>;
```

Comment:
this is also a breaking change, `IControlledProps` is public API. it's used less often than `IInputGroupProps`, but nonetheless has the potential to cause a break. you should probably do `IControlledProps2`

Reply:
Done